lib/git-subrepo

```diff
@@ -1458,7 +1458,8 @@ assert-subdir-empty() {
 get-all-subrepos() {
   local paths=();
   paths=($(
-    find . -name '.gitrepo' |
+    git ls-files |
+      grep '.gitrepo$' |
       grep -v '/.git/' |
```

Comment:
@jamezpolley ``grep -v '/.git/'`` should not be needed when switching to ``git ls-files``

Reply:
Thanks!